release_notes/migrating_to_0.20.md

```diff
@@ -1,4 +1,4 @@
-# JWT v2 Compatibility
+# NGS CLI v 0.20.0
 
 Version 0.20 brings compatibility with JWT v2 functionality added to the NATS ecosystem. Future updates to NGS will introduce features like JetStream which will require version 2 if not deployed as a leaf node.
 
```

Comment:
clarify '...which will require version 2...' Which should be version 2, NGS CLI or JWT v2?